### DISCUSSION TITLE
Changing search from an exact phase match to matching all keywords

### DISCUSSION THREAD

**Discussion by webwake** [massCodeIO/massCode#443] [Ideas]
Would like to see the search work similar to a google search, in that it is simply looking for all of the keywords (regardless of where they occur in the snippet) and not the search as an exact phrase.

If you wanted to do an exact phase search you could wrap it with quotes "Search This"

**Comment by antonreshetov** [comment]
@webwake, Thank you for the idea. https://github.com/massCodeIO/massCode/pull/446

**Comment by langrock** [comment]
Automatically scrolling to the occurrence of the search term within a note would also be useful once a note has been selected. I think that nvAlt and Simplenote do a good job at this.
